showcase/templates/edit_showcase.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'base.html' %}
+{% load static %}
+
+{% block content %}
+<form method="POST" enctype="multipart/form-data">
+    {% csrf_token %}
+    <input type="hidden" name="display_name" value="{{project.display_name}}" />
+    <input type="hidden" name="created_by" value="{{request.user.id}}" />
```

Comment:
`CTRL+H` find and replace any ` />` with just `>`.
Use the 🔍 on your IDE to find any files with ` />`. 😆

Reply:
Removed as suggested.